### PR TITLE
Remove theme switcher + add new colors

### LINKED ISSUE #429: Remove theme switch

### LINKED ISSUE #437: Add new theme (hybrid dark / light) throughout

### PR DESCRIPTION
**Pull request by lsaether**
closes #429 
closes #437

### CODE REVIEW

File: components/drawer/LeftDrawer.tsx [hunk 1]
```diff
@@ -19,22 +18,21 @@ const LeftDrawer = observer(() => {
...
-      <div className="w-full flex flex-col h-full bg-sky-100 dark:bg-black text-sky-600">
+      <div className="w-full flex flex-col h-full bg-fog-of-war dark:bg-black text-sky-600">
         <div
           className="flex mx-ztg-30 mt-ztg-20 h-ztg-35 mb-ztg-4"
           onClick={handleHomeClick}
           role="button"
         >
           <Logo />
           <div
-            className={`flex flex-col text-white  ml-ztg-14  ${
```
Comment (Robiquet) [inline components/drawer/LeftDrawer.tsx:30]: Are you using the project prettier config? Seems like there are a few formatting changes

File: components/menu/index.tsx [hunk 2]
```diff
@@ -38,68 +33,37 @@ const Menu: FC = observer(() => {
...
-          }}
-        >
-          {activityItem.label}
-        </MenuItem> */}
+        {
+          Object.keys(navigationStore.items).map((itemKey, idx) => {
+            const item = navigationStore.items[itemKey];
+
+            // The top item should have a marginTop property.
+            const className = idx === 0 ? "mt-ztg-32 mb-ztg-12" : "mb-ztg-12";
```
Comment (Robiquet) [inline components/menu/index.tsx:41]: Think it's simpler to have `mb-ztg-12` as a class on `MenuItem` and `mb-ztg-32` on the div above

**Review by Robiquet — COMMENTED** [review]
Think hover could should be adjusted on the menu here 
image

File: components/drawer/LeftDrawer.tsx (hunk 1 above)
Comment (Robiquet) [reply components/drawer/LeftDrawer.tsx:30]: That's not how it's being formatted on my side. If you click this in the bottom right:
image
Do the prettier info logs match up with the `.prettierrc.js` file like this:
image

Comment (lsaether) [reply components/drawer/LeftDrawer.tsx]: fixed in [b7487cc](https://github.com/zeitgeistpm/ui/pull/441/commits/b7487cc5e58a2e52939b495f43fe110956a99fbe)

File: components/menu/index.tsx (hunk 2 above)
Comment (lsaether) [reply components/menu/index.tsx]: fixed [289be98](https://github.com/zeitgeistpm/ui/pull/441/commits/289be9872641755e66e2cba16adf978547d967a8)

**Review by Robiquet — APPROVED** [review]
Thanks!

**Pull request merged**